### ISSUE TITLE
Travis failing

### ISSUE THREAD

**Issue by tony** [django-polymorphic/django-polymorphic#331]
It's difficult to get a proper feedback loop going if CI isn't update

Even with the benefit of the admin typo fix at #330, [tests still fail](https://travis-ci.org/django-polymorphic/django-polymorphic/builds/312453794).

This needs to get squared away so new contributions can be verified

**Comment by tony** [comment]
```
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
  File "/Users/me/work/python/django-polymorphic/.venv/lib/python3.6/site-packages/django/contrib/admin/options.py", line 615, in get_form
    fields = flatten_fieldsets(self.get_fieldsets(request, obj))
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 191, in get_fieldsets
    other_fields = self.get_subclass_fields(request, obj)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 209, in get_subclass_fields
    form = self.get_form(request, obj, exclude=exclude)
  File "/Users/me/work/python/django-polymorphic/polymorphic/admin/childadmin.py", line 65, in get_form
    return super(PolymorphicChildModelAdmin, self).get_form(request, obj, **kwargs)
RecursionError: maximum recursion depth exceeded in comparison
```

**Comment by tony** [comment]
It seems to have happened here: https://github.com/django-polymorphic/django-polymorphic/commit/02b71d8ba9b730aaf78b7775b968d26978ac77e8

https://travis-ci.org/django-polymorphic/django-polymorphic/builds

**Comment by tony** [comment]
See #322

**Comment by vdboor** [comment]
Fixed with merging #322
